content/dev-guide/_index.md

```diff
@@ -0,0 +1,21 @@
++++
+title = "Developer Guide"
+date = 2023-10-27T13:44:04-07:00
+weight = 2
++++
+
```

Comment:
Also I don't know why these paragraphs ended up in an index file, is there a reason for that?

Reply:
Because it is the first page of the guide.